src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php

```diff
@@ -192,6 +197,11 @@ private function getBundleHierarchy(ContainerBuilder $container)
             }
             $container->addResource(new FileExistenceResource($dir));
 
+            if (file_exists($dir = $container->getParameterBag()->resolveValue($config['default_path']).'/overrides/'.$name)) {
```

Comment:
cant we just move the sub path (`/overrides/`) to the config layer? Id say `%kernel.project_dir%/templates/bundles` is a sane default.

Not sure why you prefer "overrides" (to vague to me).